en/04_Changelogs/5.0.0.md

```diff
@@ -26,9 +35,67 @@ title: 5.0.0 (unreleased)
   - [Other features](#other-features)
 - [Bugfixes](#bugfixes)
 
+## Change to supported modules {#supported-modules}
+
+Some Silverstripe CMS modules are commercially supported. Silverstripe commits to looking after those modules for the duration of the Silverstripe CMS 5 lifecycle.
```

Comment:
```suggestion
Some Silverstripe CMS modules are commercially supported. Silverstripe Ltd. commits to looking after those modules for the duration of the Silverstripe CMS 5 lifecycle.
```

Reply:
Our internal style guide calls to avoid  "Silverstripe ltd". Honestly, I think calling the company "Silverstripe" was a mistake, but I'm about 15 years too late to this particular argument.

Reply:
But we can still revive the Silverstripe Foundation idea :)